2-mandatory/1-writers.js

```diff
@@ -81,38 +95,49 @@ Exercise 3:
 */
 
 function logAliveWritersInTheirForties() {
-  // write your code here
+  let arr = writers;
+  let result = arr.map((e) => {
+    e.age >= 40 && e.age <= 49 && e.alive === true
+      ? console.log(
+          `Hi, my name is ${e.firstName} ${e.lastName}. I am ${e.age} years old.`
+        )
+      : null;
+  });
+  return result;
```

Comment:
This works well. You don't need to reassign "writers" to "arr". you can call map directly on "writers", unless you see some advantage to doing it your way.